browser/ephemeral_storage/ephemeral_storage_browsertest.cc

```diff
@@ -449,3 +450,56 @@ IN_PROC_BROWSER_TEST_F(EphemeralStorageBrowserTest,
   EXPECT_EQ("", private_values.iframe_1.cookies);
   EXPECT_EQ("", private_values.iframe_2.cookies);
 }
+
+IN_PROC_BROWSER_TEST_F(EphemeralStorageBrowserTest,
+                       NavigationCookiesArePartitioned) {
+  AllowAllCookies();
```

Comment:
do we have a ticket for connecting this to block3p shield setting?

Reply:
Can you confirm whether or not one of these tickets is what you are referring to?
 * https://github.com/brave/brave-browser/issues/12789
 * https://github.com/brave/brave-browser/issues/12790

I believe that the first bug covers the situation where cookies are blocked in brave shields.